vm-run

```diff
@@ -33,7 +33,8 @@ parser.add_argument('-G', '--graphics', action='store_true', help='Display a gra
 parser.add_argument('-q', '--quiet', action='store_true', help="Don't connect text console")
 parser.add_argument('-c', '--client', action='store_true', help="Open Cockpit Client (implies -q)")
 parser.add_argument('-C', '--cpus', default=None, type=int, help='Number of cpus in the target machine')
-parser.add_argument('-S', '--storage', default=None, type=str, help='Add a second qcow2 disk at this path')
+parser.add_argument('-S', '--storage', type=str, action='append',
+                    help='Add a qcow2 disk(s) at this path')
```

Comment:
Drop the `type=str` (it's the default anyway), and instead add `default=[]`. Then...

Reply:
While you are at it, `default=None` is pointless, you can also drop it from all other options. Perhaps do a first commit of dropping type=str and default=None?

Reply:
Good idea!